### PR TITLE
use local_ip OR hostName represent graph in meta

### PR DESCRIPTION
**Pull request by nevermore3**
https://jira.nebula-graph.io/browse/NG-456

### CODE REVIEW

File: src/service/GraphFlags.cpp [hunk 1]
```diff
@@ -23,6 +23,7 @@ DEFINE_int32(num_worker_threads, 0, "Number of threads to execute user queries")
 DEFINE_bool(reuse_port, true, "Whether to turn on the SO_REUSEPORT option");
 DEFINE_int32(listen_backlog, 1024, "Backlog of the listen socket");
 DEFINE_string(listen_netdev, "any", "The network device to listen on");
```
Comment (Shylock-Hg) [inline src/service/GraphFlags.cpp:25]: This option is not used.

**Review by dutor — APPROVED** [review]
LGTM.

**Review by jievince — APPROVED** [review]

**Review by Shylock-Hg — DISMISSED** [review]

**Review by yixinglu — COMMENTED** [review]
please check the flag local_ip in nebula-graphd.conf

**Review by nevermore3 — COMMENTED** [review]
> please check the flag local_ip in nebula-graphd.conf

has exist in nebula-graphd.conf

**Review by yixinglu — APPROVED** [review]

**Review by CPWstatic — APPROVED** [review]

**Pull request merged**